output/typescript/types.ts

```diff
@@ -12833,9 +12833,9 @@ export interface SecurityClearCachedRealmsRequest extends RequestBase {
 }
 
 export interface SecurityClearCachedRealmsResponse {
+  _nodes: NodeStatistics
```

Comment:
Missing `@identififer`?

Reply:
No, this is the TypeScript generator output. This diff is caused by the reordering of some `_nodes` properties that were last in the list to the first position and which I moved at the top so that all uses are consistent.

I was also confused by this while reading the diff before creating the PR :-)